component.yaml

```diff
@@ -5,14 +5,13 @@ description: "Install Tuist and Generate a Project."
 inputs:
 - key: "AC_TUIST_PATH"
   defaultValue: "$AC_REPOSITORY_DIR"
-  isRequired: false
+  isRequired: true
   title: Project Path
-  description: The path to the directory that contains the definition of the project. Defaults to current directory.
-- key: "AC_TUIST_PROJECT_ONLY"
-  defaultValue: "false"
+  description: The path to the directory of Tuist project. By default, it is set to the root directory of the cloned repository.
+- key: "AC_TUIST_VERSION"
```

Comment:
IMO there should be two fields below as input for React and Flutter project to work:
- title: Repository Directory
defaultValue: "$AC_REPOSITORY_DIR"

- title: Project Path
defaultValue: "$AC_PROJECT_DIR"

Reply:
It will be resolved by 
> ... this component should specify the "buildPlatform: ObjectiveCSwift"

Details [here](https://linear.app/appcircle/issue/BE-4430/tuist-ios-component-improvement#comment-589bfb90).

Reply:
@csonuryilmaz I changed the name and build platform type of component.